modules/protocol/chatircd1.1.c

```diff
@@ -0,0 +1,352 @@
+/*
```

Comment:
There seems quite a lot of code duplication from protocol/charybdis.c here. That can be changed similar to the ircd-seven module (rather than ts6-generic, depend on charybdis in `_modinit`, then update what only what needs to be changed).

Reply:
No longer a relevant concern, as the protocol module has been rewritten to resolve this.